README.md

```diff
@@ -10,21 +10,22 @@ pip install xbmc-client
 Available options :
 
 ```sh
-Usage: xbmc-client [options]
+Usage: xbmc_client.py [options]
```

Comment:
keeping `xbmc-client` is legit as it is the [name of the entrypoint](https://github.com/jcsaaddupuy/xbmc-client/blob/2f5c978959969554b00fffc85e282c093c73bde8/setup.py#L20) once installed